src/test/java/bit/javaoop/EmployeeTest.java

```diff
@@ -2,17 +2,49 @@
 
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class EmployeeTest {
 
     @Test
-    void shouldBeSatisfied() {
-        //given
+    void isSatisfiedTest() {
+
         Employee highSalaryEmployee = new Employee("Bogaty pracownik", 50_000);
-        //when
+
         boolean result = highSalaryEmployee.isSatisfied();
-        //then
+
         assertTrue(result);
     }
+
+    @Test
+    void getSalaryTest() {
```

Comment:
To jest to dlaczego zawsze testy wydawały mi się bezsensowne :) Pytałem się siebie dlaczego mam testować, coś oczywistego? No i w tym wypadku faktycznie nie trzeba tego testować. Nie musisz testować czegoś co jest po prostu pewne. Raczej wiadomo, że gettery zawsze będą zwracały dobrą wartość.

Chociaż test cover masz perfekcyjny. Teoretycznie powinno się testować każdą linijkę kodu, więc również i gettery. Jeśli kiedyś dojdzie do nich jakaś nieprzewidziana funkcjonalność, to być może ten test uratuje cały system :D

W praktyce jednak się po prostu tego nie robi :)